poros/game_results.rb

```diff
@@ -0,0 +1,64 @@
+
+class GameResults
+  
+  def service 
+    GameService.new
+  end
+
+  def game_params_by_name(game_name)
+    params = service.get_games_by_name(game_name)[0]
+    binding.pry
+    result = {data: {
+      age_ratings:  run_array(:get_age_ratings, :rating, params[:age_ratings]), # array of ids
+      release_date: service.get_release_date(params[:first_release_date]), # id
+      cover:        cover_url(params[:cover]),
+      popularity:   params[:popularity], # good
+      summary:      params[:summary], # good
+      name:         params[:name], # good
+      total_rating: params[:total_rating], # good
+      categories:   params[:category], # got 0
+      genres:       run_array(:get_game_genres, :name, params[:genres]), # array of ids
+      keywords:     params[:keywords], # array of ids -- several hundred :(
+      platforms:    run_array(:get_platforms, :name, params[:platforms]), # array of ids
+      similars:     run_array(:get_games_by_id, :name, params[:similar_games]), # array of ids
+      screenshots:  run_array(:get_screenshots, :url, params[:screenshots]), # array of ids
+      themes:       run_array(:get_game_themes, :name, params[:themes]), # array of ids
+      videos:       youtube_id(params[:videos][0]) # good -> array of ids
```

Comment:
Things like youtube_id only return one value, and we can hit the API (again) to return the video_id if we pass in the `video_id` (given by the api)

Reply:
this is very helpful to know